backends/qualcomm/runtime/backends/QnnBackendCache.cpp

```diff
@@ -126,13 +109,16 @@ Error QnnBackendCache::Configure() {
   Error status = GetQnnGraphInfoFromBinary();
   if (status == Error::Internal) {
     // check if context binary came from flatbuffer
-    flatbuffers::FlatBufferBuilder builder;
     flatbuffers::Verifier verifier(
         static_cast<const uint8_t* const>(qnn_context_blob_.buffer),
         qnn_context_blob_.nbytes);
 
-    if (qcir::VerifyGraphBuffer(verifier)) {
+    if (qcir::VerifyContextBuffer(verifier)) {
```

Comment:
Trying to follow - are these logic for AOT or runtime?

Reply:
If for `online_prepare`, this will happen in runtime.
If for `multiple_graphs` this will happen in AoT, since we will again compile the merged `qcir` in host side with higher optimization level in HTP.